### DISCUSSION TITLE
Attempting testing of Xbox Series S Compatibality

### DISCUSSION THREAD

**Discussion by cryingcavecat** [dmunozv04/iSponsorBlockTV#95] [General]
So to preface as soon as I saw this project (on Hacker News) I instantly starred it. I've always been vehemently anti-advert. I'm running a dual Pi-Hole setup on my home network and every computer I have uses uBlock Origin in the browser. Having been plagued with no solution for Xbox YouTube App ads, this looked promising. 

As I saw no verification of comparability, I thought I'd be the first to embark on it. 

However, I have not been able to get it to work. Things I've tried:

- Running on ARM64 Raspberry Pi from source code
- Running on AMD64 Pop!_OS Laptop from source code
- Running in Docker on AMD64 Pop!_OS Laptop
- DLNA is enabled on Xbox Series S
- All devices are on the same network

The result is that no device is detected on my network (even with the Xbox on and the YouTube App open). 

I haven't been able to test on any other devices yet since I use my Smart TV as a dumb device and thus can't rule out my setup or configuration as the fail point. Thus I've opened this as a discussion rather than a pull request denying its compatibility. I will keep trying in other configurations as they come to mind and will update here if I succeed. 

Thanks for this project :)

**Comment by dmunozv04** [comment]
If auto discovery isn’t working (it seems to only work on some devices), you can use manual linking with a TV code to link your console. 
Manual linking can be accessed from the configurator

**Reply by cryingcavecat** [reply]
Thanks for the response. I'll try that and report back.
